clients/lsp-kotlin.el

```diff
@@ -181,6 +183,61 @@ to Kotlin."
    (t (setq-local lsp-lens-backends (delete #'lsp-kotlin-lens-backend lsp-lens-backends)))))
 
 
+;; Stolen from lsp-java:
+;; https://github.com/emacs-lsp/lsp-java/blob/a1aff851bcf4f397f2a968557d213db1fede0c8a/lsp-java.el#L1065
+(declare-function helm-make-source "ext:helm-source")
+(defvar lsp-kotlin--helm-result nil)
+(defun lsp-kotlin--completing-read-multiple (message items initial-selection)
+  (if (functionp 'helm)
+      (progn
+        (require 'helm-source)
+        (helm :sources (helm-make-source
+                           message 'helm-source-sync :candidates items
+                           :action '(("Identity" lambda (_)
+                                      (setq lsp-kotlin--helm-result (helm-marked-candidates)))))
+              :buffer "*lsp-kotlin select*"
+              :prompt message)
+        lsp-kotlin--helm-result)
+    (if (functionp 'ivy-read)
+        (let (result)
+          (ivy-read message (mapcar #'car items)
+                    :action (lambda (c) (setq result (list (cdr (assoc c items)))))
+                    :multi-action
+                    (lambda (canditates)
+                      (setq result (mapcar (lambda (c) (cdr (assoc c items))) canditates))))
+          result)
+      (let ((deps initial-selection) dep)
+        (while (setq dep (cl-rest (lsp--completing-read
+                                   (if deps
+                                       (format "%s (selected %s): " message (length deps))
+                                     (concat message ": "))
+                                   items
+                                   (-lambda ((name . id))
+                                     (if (-contains? deps id)
+                                         (concat name " ✓")
+                                       name)))))
+          (if (-contains? deps dep)
+              (setq deps (remove dep deps))
+            (cl-pushnew dep deps)))
+        deps))))
+
+(defun lsp-kotlin-implement-member ()
+  (interactive)
+  (lsp-request-async
+   "kotlin/overrideMember"
+   (list :textDocument (list :uri (lsp--buffer-uri))
+         :position (lsp--cur-position))
+   (lambda (member-options)
+     (-if-let* ((option-items (-map (lambda (x)
+                                      (list (lsp-get x :title)
+                                            (ht-get (lsp-get (lsp-get x :edit)
```

Comment:
`ht-get` -> `lsp-get`

Reply:
The problem with that is that `lsp-get` seems to expect a symbol as the key, while `ht-get` allows strings as well (`(lsp--buffer-ui)` is a string). I get the following error when using `lsp-get` in place of `ht-get` here:
`Error processing message (wrong-type-argument symbolp "file:///....")`


Tried a few other solutions as well, like converting to symbol, but I couldn't get it to work properly. Let me know if there is something I'm missing 🙂

Reply:
can you try something like this:

```
(let ((key "uri"))
  (intern (concat ":" key) ))
```

The issue is that when you run that code with lsp-use-plists that map wont the hashtable but it will be plists and it won't work.